templates/typescript/events/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+builders:
+  default: paketobuildpacks/builder:base
+  base: paketobuildpacks/builder:base
+  full: paketobuildpacks/builder:full
+
+buildpacks:
+  - paketo-buildpacks/nodejs
+  - docker.io/lanceball/boson-typescript # github.com/lance/buildpacks
```

Comment:
Use the paketo buildpacks to install Node.js, npm, and function dependencies. Then use the boson-typescript buildpack to do the TS>JS transpilation.